code/modules/telesci/gps.dm

```diff
@@ -61,33 +63,39 @@ var/list/SPS_list = list()
 	else
 		..()
 
+/obj/item/device/gps/proc/get_location_name()
+	var/turf/device_turf = get_turf(src)
+	var/area/device_area = get_area(src)
+	if(emped)
+		return "ERROR"
+	else if(!device_turf || !device_area)
+		return "UNKNOWN"
+	else if(device_turf.z > WORLD_X_OFFSET.len)
+		return "[format_text(device_area.name)] (UNKNOWN, UNKNOWN, UNKNOWN)"
+	else
+		return "[format_text(device_area.name)] ([device_turf.x-WORLD_X_OFFSET[device_turf.z]], [device_turf.y-WORLD_Y_OFFSET[device_turf.z]], [device_turf.z])"
+
+
 /obj/item/device/gps/ui_interact(mob/user, ui_key = "main", var/datum/nanoui/ui = null, var/force_open=NANOUI_FOCUS)
 	var/data[0]
 	if(emped)
 		data["emped"] = TRUE
 	else
 		data["gpstag"] = gpstag
 		data["autorefresh"] = autorefreshing
+		data["location_text"] = get_location_name()
+		data["unique_id"] = unique_id
```

Comment:
You're never using this in the template, why is this here?

Reply:
location_text is used https://github.com/vgstation-coders/vgstation13/pull/18242#discussion-diff-187188479R27

Reply:
Yes, I was referring to unique_id